docs/specs/om/open_metrics_spec_2_0.md

```diff
@@ -765,15 +887,72 @@ foo_sum 324789.3
 foo_created 1520430000.123
 ```
 
+##### Histogram with exponential buckets
+
+The MetricPoint's value MUST be a complex data type.
+
+Histograms with exponential buckets use the integer native histogram data type.
+
+The integer native histogram data type is a JSON like structure with fields. There MUST NOT be any whitespace around fields.
```

Comment:
It's not really JSON. Many language use vaguely this kind of structure. I wouldn't call out JSON because it creates the expectation that other JSON syntax rules also apply.

Reply:
> There MUST NOT be any whitespace around fields.

Has it already been decided that OMv2 will stay whitespace intolerant?

It's one of my big big red flags with OMv1 that it doesn't tolerate whitespace like the classic Prometheus text format. If OMv2 will see the light and allow whitespace again, then of course we should allow whitespace here.

Reply:
I'd like to defer this to a separate PR.

Reply:
I resolve this to mean "I have made my point". I understand that the actual decision has been deferred. (Should OMv2 decide to be whitespace tolerant (which I hope), then I would argue that the representation of native histograms should be whitespace tolerant, too.)